src/styles/section.scss

```diff
@@ -71,3 +71,7 @@
   bottom: 0px;
   transform: translate(-50%, 50%);
 }
+
+.ob-section__collapsed {
+  display: none;
+}
```

Comment:
What is the CSS here for?

Reply:
To make anything inside a collapsed section actually return false from the `element.checkVisibilty()` method, so we can determine that an element is in a collapsed section.

Reply:
Sweet makes sense